code/game/objects/items/pneumaticCannon.dm

```diff
@@ -93,13 +102,7 @@
 	if(needs_air == FALSE)
 		return
 	playsound(src, 'sound/items/ratchet.ogg', 50, TRUE)
-	switch(pressureSetting)
-		if(LOW_PRESSURE)
-			pressureSetting = MID_PRESSURE
-		if(MID_PRESSURE)
-			pressureSetting = HIGH_PRESSURE
-		if(HIGH_PRESSURE)
-			pressureSetting = LOW_PRESSURE
+	pressureSetting = pressureSetting >= HIGH_PRESSURE ? LOW_PRESSURE : pressureSetting + 1
 	balloon_alert(user, "output level set to [pressureSetting]")
```

Comment:
You didn't change this

Reply:
Can you explain

Reply:
You're not answering so i'll just set it on ready for review

Reply:
This still gives the number and not the text

Reply:
Uhh, i don't quite understand why do you need the text outside of examine proc, and it changes it's regimes pretty fine:
![image](https://user-images.githubusercontent.com/93882977/225552908-646ea0e6-4956-4201-b78f-83bb3ee11e17.png)
I'm afraid i don't see an issue in point blank distance.
Also powerfist has same code for changing pressure settings so yeah.

Reply:
This is the balloon alert. What does the balloon alert look like